package.json

```diff
@@ -44,13 +44,29 @@
             {
                 "command": "aks.periscope",
                 "title": "Run AKS Periscope"
+            },
+            {
+                "command": "azure-deploy.configureCicdPipeline",
+                "title": "Configure CI/CD"
```

Comment:
This is going out to all AKS users, not just people who choose to install the CI/CD pipelines extension.  So it needs consistent and explanatory language.  I don't know enough about the feature to be sure what to suggest, but maybe `Configure CI/CD Pipelines` and `Browse CI/CD Pipelines`.  Possibly even go as far as `Configure Azure CI/CD Pipelines` because all _my_ CI/CD pipelines are on GitHub and I would be puzzled if the AKS extension thought it knew about them!

Reply:
@pulkitaggarwl please let me know what will the updated strings be

Reply:
Sure we can be more descriptive. we can use

- Configure CI/CD Workflow
- Browse CI/CD Workflows

@itowlson how does these sound ?